opencensus/trace/propagation/binary_format.py

```diff
@@ -108,7 +108,7 @@ def from_header(self, binary):
         # from binary.
         try:
             data = Header._make(struct.unpack(BINARY_FORMAT, binary))
-        except struct.error:
+        except Exception:
```

Comment:
What other types of exceptions were you seeing?

Reply:
Added check for None input and changed the exception type back to `struct.error`.